.travis.yml

```diff
@@ -124,6 +124,7 @@ env:
     - TRAVIS_FLAVOR=twemproxy FLAVOR_VERSION=latest
     - TRAVIS_FLAVOR=varnish FLAVOR_VERSION=4.1.7
     - TRAVIS_FLAVOR=varnish FLAVOR_VERSION=5.0.0
+    - TRAVIS_FLAVOR=fargate FLAVOR_VERSION=latest
```

Comment:
this needs to be renamed to ecs_fargate

Reply:
went a bit quick on the folder rename 🤦‍♂️

doing all the changes